EKF/estimator_base.h

```diff
@@ -58,6 +58,33 @@ struct gps_message {
 	bool vel_ned_valid;		// GPS ground speed is valid
 };
 
+struct parameters {
+	float mag_delay_ms;
```

Comment:
They should be all initialized to sane defaults. Try if you can do that here, C++11 added support for some of that.